client/shared/src/components/FileMatchChildren.tsx

```diff
@@ -124,14 +152,16 @@ export const FileMatchChildren: React.FunctionComponent<FileMatchProps> = props
                             }`}
                             className={classNames('test-file-match-children-item-wrapper', styles.itemCodeWrapper)}
                         >
-                            <Link
-                                to={createCodeExcerptLink(group)}
+                            <Link to={createCodeExcerptLink(group)} aria-label="Open file match" />
```

Comment:
very minor: "File match" seems like an odd wording to me

```suggestion
                            <Link to={createCodeExcerptLink(group)} aria-label="Open search result" />
```